test/mix/sentry.package_source_code_test.exs

```diff
@@ -20,15 +20,26 @@ defmodule Mix.Tasks.Sentry.PackageSourceCodeTest do
 
     assert :ok = Mix.Task.rerun("sentry.package_source_code")
 
-    assert_receive {:mix_shell, :info, ["Wrote " <> _ = message]}
-    assert message =~ expected_path
+    validate_map_file!(expected_path)
+  end
 
-    assert {:ok, contents} = File.read(expected_path)
+  test "packages source code into custom path" do
+    put_test_config(
+      root_source_code_paths: [File.cwd!()],
+      enable_source_code_context: true
+    )
 
-    assert %{"version" => 1, "files_map" => source_map} =
-             :erlang.binary_to_term(contents, [:safe])
+    expected_path =
+      [System.tmp_dir!(), "sentry.map"]
+      |> Path.join()
```

Comment:
Instead of this and then the `try`/`after`, you can use the `@tag :tmp_dir` ExUnit tag and do `expected_path = Path.join(tmp_dir, "sentry.map")`, without having to remove the file after the test.